packages/chat-component-bindings/src/messageThreadSelector.ts

```diff
@@ -136,11 +136,15 @@ const hasValidParticipant = (chatMessage: ChatMessageWithStatus): boolean =>
 export const messageThreadSelector: MessageThreadSelector = createSelector(
   [getUserId, getChatMessages, getLatestReadTime, getIsLargeGroup, getReadReceipts, getParticipants],
   (userId, chatMessages, latestReadTime, isLargeGroup, readReceipts, participants) => {
+    // We can't get displayName in teams meeting interop for now, disable rr feature when it is teams interop
+    const isTeamsInterop = Object.values(participants).find((p) => 'microsoftTeamsUserId' in p.id) !== undefined;
+
     // get number of participants
     // filter out the non valid participants (no display name)
-    const messageThreadParticipantCount = Object.values(participants).filter(
-      (p) => p.displayName && p.displayName !== ''
-    ).length;
+    // Read Receipt details will be disabled when participant count is 0
+    const messageThreadParticipantCount = isTeamsInterop
+      ? 0
+      : Object.values(participants).filter((p) => p.displayName && p.displayName !== '').length;
 
```

Comment:
Further down on L178 the `readBy` will be incorrect as this also filters on `.name !== ''` inside `getParticipantsWhoHaveReadMessage`, this should also be set to undefined or 0.

Reply:
Updated:D will update the cherry pick branch after merging this in